extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts

```diff
@@ -14,15 +14,128 @@ import { Ui } from '../../../js/common/browser/ui.js';
 import { Xss } from '../../../js/common/platform/xss.js';
 import { MsgBlockParser } from '../../../js/common/core/msg-block-parser.js';
 import { AcctStore } from '../../../js/common/platform/store/acct-store.js';
+import { GmailParser } from '../../../js/common/api/email-provider/gmail/gmail-parser.js';
+import { Str } from '../../../js/common/core/common.js';
 
 export class PgpBlockViewRenderModule {
   public doNotSetStateAsReadyYet = false;
 
   private heightHist: number[] = [];
+  private printMailInfoHtml!: string;
 
   constructor(private view: PgpBlockView) {
   }
 
+  public initPrintView = async () => {
+    const fullName = await AcctStore.get(this.view.acctEmail, ['full_name']);
+    Xss.sanitizeRender('.print_user_email', `<b>${fullName.full_name}</b> &lt;${this.view.acctEmail}&gt;`);
+    try {
+      const gmailMsg = await this.view.gmail.msgGet(this.view.msgId!, 'full', undefined);
```

Comment:
I have several concerns about this call. Perhaps, some of them can be addressed in a separate issue.
1) Do we want to ignore the `canReadEmails` "setting" (calculated in `PgpBlockViewDecryptModule`)? @tomholub Can you give us some background why this setting had appeared? Was it to prevent fetching messages when an Outlook mail site is opened or for some limited permission installations?
2) If we only need headers, wouldn't it be enough to fetch `metadata` instead of `full` message?
3) It'd be better to do the fetching in `PgpBlockViewDecryptModule`, as it sometimes fetches the message by `msgId`, sometimes just takes it from a URL object (`encryptedMsgUrlParam`), and it's probably going to be more complicated as with https://github.com/FlowCrypt/flowcrypt-browser/issues/4552#issuecomment-1207852190 we may need to try and fetch the message if reading the URL object fails. It seems the `PgpBlockViewDecryptModule` is more competent about what message data is already fetched and what is missing. And `PgpBlockViewRenderModule` would be limited to rendering only.

Reply:
`canReadEmails` can be ignored, see https://github.com/FlowCrypt/flowcrypt-browser/issues/4600

Reply:
indeed `metadata` should work. (btw `full` still skips attachments so it's not the worst)

Reply:
Agree with third point. Sometimes PgpBlockViewDecryptModule already downloads the message so no need to do it twice. We could make it always download the message, for example (but in a way that offline viewing of encrypted messages still works)

Reply:
@rrrooommmaaa Thank you for your notice.
Just changed `full` to `meta`
Will fix getting message twice issue(which we do in both render and decrypt module) in #4600